.circleci/config.yml

```diff
@@ -373,15 +373,28 @@ jobs:
           working_directory: ./sqrl-testing/sqrl-integration-tests/src/test/resources/usecases/udf
           name: Launch sqrl-server and test if API is running
           command: |
-            docker run --rm -v "$PWD:/build" -e TZ="UTC" datasqrl/cmd-test-image:${CIRCLE_SHA1} compile myudf.sqrl
-            docker run -d --name sqrl-server --rm -v $(pwd)/build/deploy/plan/vertx.json:/opt/sqrl/server-model.json \
-                                                  -v $(pwd)/build/deploy/plan/vertx-config.json:/opt/sqrl/server-config.json \
+            docker run -v "$PWD:/build" -e TZ="UTC" datasqrl/cmd-test-image:${CIRCLE_SHA1} compile myudf.sqrl
+            docker run -d --name sqrl-server -v $(pwd)/build/deploy/plan/vertx.json:/opt/sqrl/vertx.json \
+                                                  -v $(pwd)/build/deploy/plan/vertx-config.json:/opt/sqrl/vertx-config.json \
                                                   -p 8888:8888 \
                                                   datasqrl/sqrl-server-test-image:${CIRCLE_SHA1}
             sleep 2s
             docker logs sqrl-server
             curl -s -X POST http://localhost:8888/graphql   -H 'Content-Type: application/json'   --data '{"query":"query { __typename }"}'
 
+  run-jwt-tests:
+    executor: default-machine
+    steps:
+      - checkout
+      - run:
+          name: Pull test image
+          command: |
+            docker pull datasqrl/cmd-test-image:${CIRCLE_SHA1}
+            docker pull datasqrl/sqrl-server-test-image:${CIRCLE_SHA1}
+      - run:
+          name: Run JWT tests
+          command: ./.circleci/run-jwt-tests.sh
```

Comment:
We need a better approach to run this final testing on the images....

is important to test the images after they are backed, but need something better structured than bash files